src/rules.js

```diff
@@ -45,3 +45,16 @@ export function RequiredFields(context, options) {
     },
   };
 }
+
+export function typeNamesShouldBeCapitalized(context) {
+  return {
+    NamedType(node) {
```

Comment:
does `NamedType` get called for inline fragments? i.e., can we add a test case for
```graphql
query {
  someUnion {
    ... on someUnionMember {
      someField
    }
  }
}
```